README.md

```diff
@@ -147,6 +148,41 @@ resource "docker_container" "dev" {
 }
 ```
 
+### PrivateKey-based authentication
+* Prebuild private key in your envbuilder image
```

Comment:
Isn't this dangerous if we use a cache registry ?
The layer containing the private key may end up in the registry ?

Reply:
My original assumption was that this image may be stored in a private registry, so I didn't take into consideration the potential issue of having a private key in the image layers.